python/Ganga/Core/MonitoringComponent/Local_GangaMC_Service.py

```diff
@@ -1162,6 +1162,8 @@ def credChecker():
             except Exception as msg:
                 return False
             else:
+                if s is None:
+                    return True
```

Comment:
Thanks for confirming the design. I'm assuming the old renew methods are being removed in this case?